packages/suite/src/views/wallet/coinmarket/common/AddressOptions.tsx

```diff
@@ -87,6 +89,9 @@ export const AddressOptions = <TFieldValues extends AddressOptionsFormState>({
     const addresses = account?.addresses;
     const addressDictionary = useAccountAddressDictionary(account);
     const value = address ? addressDictionary[address] : undefined;
+    const accountMetadata = useSelector(
+        state => account && selectLabelingDataForAccount(state, account.key),
+    );
```

Comment:
```suggestion
    const accountMetadata = useSelector(
        state => selectLabelingDataForAccount(state, account?.key),
    );
```

If this component can be rendered without `account`
I believe it would be better to modify `selectLabelingDataForAccount` so that it accepts `undefined` as a key and handle it inside the method instead of this conditional selector

Reply:
Another option is to simply pass `account?.key || ''`

Reply:
if you are about to make changes to selector, please consider changes in selectors in this branch https://github.com/trezor/trezor-suite/pull/9130 so that I don't have to solve too many conflicts :D

Reply:
fixed by `account?.key || ''` as also used in other places